pkg/solana/contract.go

```diff
@@ -309,16 +309,27 @@ func GetLatestTransmission(ctx context.Context, client *rpc.Client, account sola
 	if len(t) != int(transmissionLen) { // validate length
 		return Answer{}, 0, errTransmissionLength
 	}
+	buf = bytes.NewReader(t)
```

Comment:
@aalu1418 still need to adjust this so that we read the version from the header, then if v1, read using the old offsets (24 byte transmission lengths etc), if v2 then use the following.

Maybe we can finally use https://github.com/gagliardetto/binary with a Decoder?